### ISSUE TITLE
Var declarations should be joined - Airbnb?

### ISSUE THREAD

**Issue by ekmartin** [jscs-dev/node-jscs#972]
The following snippet fails (in the online demo) with the Airbnb preset, even though the style guide states that you should "Use one var declaration per variable.", [here](https://github.com/airbnb/javascript#variables).

```js
var express = require('express');
var apiRoutes = require('./api');
var authRoutes = require('./auth');

var router = express.Router();
```

**Comment by alawatthe** [comment]
This is a duplicate of https://github.com/jscs-dev/node-jscs/pull/822.

**Comment by markelog** [comment]
Updated preset will be available in 1.11
